hassio/src/ingress-view/hassio-ingress-view.ts

```diff
@@ -26,20 +29,40 @@ class HassioIngressView extends LitElement {
 
   @internalProperty() private _addon?: HassioAddonDetails;
 
+  @property({ type: Boolean })
+  public narrow = false;
+
   protected render(): TemplateResult {
     if (!this._addon) {
       return html` <hass-loading-screen></hass-loading-screen> `;
     }
+    const inConfigPanel = location.pathname.startsWith("/hassio/ingress");
```

Comment:
Let's add a property to indicate this. This component should not be aware of what URLs it is mounted.